norfair/drawing.py

```diff
@@ -14,8 +14,10 @@ def draw_points(frame, detections, radius=None, thickness=None, color=None):
         thickness = int(max(frame_scale / 7, 1))
     if color is None:
         color = Color.red
-
+    color_is_rand = color==Color.rand
     for d in detections:
+        if color_is_rand:
+            color = Color.random(random.randint(0, 20))
```

Comment:
This should be handled by the `Color` class internally, so the randomness logic is defined next to where it is used, and the user only has to think about calling `Color.red`, `Color.blue` or `Color.random` without knowing what happens behind the scenes.